src/explorer/CosmosDBConnection.ts

```diff
@@ -53,8 +54,8 @@ export class CosmosDBConnection extends AzExtTreeItem {
         const appSettingsClient: IAppSettingsClient = this.parent.parent.client;
         const appSettings: WebSiteManagementModels.StringDictionary = await appSettingsClient.listApplicationSettings();
         if (appSettings.properties) {
-            const warning: string = `Are you sure you want to remove connection "${this.label}"? This will delete the following application settings: ${this.appSettingKeys.map((s) => `"${s}"`).join(', ')}.`;
-            await context.ui.showWarningMessage(warning, { modal: true }, DialogResponses.deleteResponse);
+            const warning: string = localize('removeConnection', 'Are you sure you want to remove connection "{0}"? This will delete the following application settings:', this.label);
+            await context.ui.showWarningMessage(`${warning} ${this.appSettingKeys.map((s) => `"${s}"`).join(', ')}`, { modal: true }, DialogResponses.deleteResponse);
```

Comment:
How come you aren't including this kind of stuff in localize? I would say we should

Reply:
Well, it should just be the keys from the appSettings.  I guess the their position in the sentence might be something that needs to be localized?

Reply:
Yes, exactly